### DISCUSSION TITLE
Letzte aktualisierung keine bewegung

### DISCUSSION THREAD

**Discussion by Kante001** [tbnobody/OpenDTU#1912] [Q&A]
Hallo,

ich habe meine OpenDTU mit ESP32, funkmodul mit Antenne und adapter und Display aktualisiert mit dem flash tool der Website auf die Version v24.1.21. vorher hatte ich noch eine Recht alte Version drauf. Vorher hat alles bestens funktioniert.
So jetzt die Frage nach dem Update habe ich die pin_mapping geändert und hochgeladen, generisches nodemcu 38 Pin mit ssd1106 1.3 genommen.
Er zeigt rot markiert eth clk_mode -1 4294967295 und Typ -1 4294967295. Display zeigt soweit an.
Wechselrichter werden erkannt.
Aber die Zeit bei letzt Aktualisierung läuft nicht wie früher hoch sondern steht statisch Rum. Und die Steuerung mit dem resperipy regelt aber die opendtu scheint es nicht an die WR zu senden. Hat jemand eine Idee ob eine Einstellung im opendtu falsch ist oder was anderes ?

Viele Grüße

**Comment by ghost** [comment]
Hallo,

Das Problem liegt sicher hier: 

!! IMPORTANT UPGRADE NOTES !!
If you are upgrading from a version before 15.03.2023 you have to upgrade the partition table of the ESP32. Please follow the [this](https://github.com/tbnobody/OpenDTU/blob/master/docs/UpgradePartition.md) documentation!

(Steht auf der Startseite mit als erstes)

**Reply by Kante001** [reply]
Hallo ich habe mit dem flash tool opendtu-generic_esp32.factory.bin geflasht aber das Problem besteht weiterhin.

**Reply by ghost** [reply]
Mal Kaltstart gemacht, also Spannung abgezogen?
Dem einen oder anderem hat das geholfen.

Ähm, alle Einstellungen sind auch wieder herzustellen.

**Reply by Kante001** [reply]
Ja Stecker hab ich paar Mal raus gemacht und Einstellungen auch so wie vorher paar Punkte sind ja dazu gekommen deswegen ja die Frage ob da irgendwas besonderes zu beachten ist

**Reply by ghost** [reply]
Sorry, da habe ich keine Idee mehr, vielleicht meldet sich noch einer anderer hier.
Das hast du bestimmt auch schon versucht: Möglich wäre noch die Werkseinstellungen zu laden. Nicht das durch die Neupartitionierung doch irgendetwas Falsches in den Einstellungen steht.

**Reply by Kante001** [reply]
Ich hab jetzt noch mal die Datei geflasht aber der will ein WLAN Passwort um sich mit der opendtu Netzwerk zu verbinden. Jetzt kommt mir in Sin das war letztens auch so und ich habe den flash mit der websiten oberfläche gemacht. Wie ist der WLAN schlüssel für das Netzwerk der Opendtu im Neuzustand ? openDTU42 geht nicht.

**Reply by ghost** [reply]
Ich wüste nicht, das sich das geändert hätte:
http://192.168.4.1/
Benutzer: admin
PW: openDTU42
Wenn sich die DTU in kein WLAN einbuchen kann, macht der Accesspoint auch nicht nach einer Weile zu. (Glaube ich gelesen zu haben)

**Reply by Kante001** [reply]
Nein ich meine wenn es neu aufgesetzt ist, öffnet die opendtu ja ein eigenes Netzwerk und mit diesem kann ich mich nicht verbinden. Er will ein Password aber openDTU42 geht nicht. Hab gerade noch mal die veraltete generic factory bin geflasht aber genau das selbe oder muss ich nach der factory noch die normale ohne factory hinter her Flaschen ?

**Reply by ghost** [reply]
Ja genau.
Die DTU macht nach dem Neustart einen Accesspoint auf.
Der WLAN Name ist in der Art: OpenDTU-XXXXXXX
 Die "X"e stehen für ein paar Ziffern der ESP Chip-ID.
Mit diesem WLAN ist sich zu verbinden.
Das Passwort ist normalerweise das selbe, wie für die Weboberfläche: 
openDTU42

Das muss funktionieren.

**Reply by ghost** [reply]
Ich bin kein Datentechniker, aber was ich mir vorstellen kann ist noch, das deine WLAN-Schnittstelle eine feste IP hat. Vielleicht sorgt das für die Login-Probleme. Setze deinen Adapter mal auf automatisch beziehen.

Was ich vergaß: Nochmal flashen musst du nicht. In der ...factory...bin ist alles drin.

**Reply by ghost** [reply]
Ich habe nochmal in die [Hilfe](https://www.opendtu.solar/) geschaut.
Ich denke wir beide haben das richtig versucht:

- Nach dem ersten Flashen des Mikrocontrollers wird ein Access Point namens „OpenDTU-*“ geöffnet. Das Standardpasswort ist openDTU42.
- Öffnen Sie mit einem Webbrowser die Adresse http://192.168.4.1
- Navigieren Sie zu Einstellungen -> Netzwerkeinstellungen und geben Sie Ihre WLAN-Anmeldeinformationen ein. Der Benutzername für den Zugriff auf das Konfigurationsmenü lautet „admin“ und das Passwort ist dasselbe wie für den Zugriff auf den Access Point (Standard: openDTU42).
- OpenDTU stellt dann gleichzeitig mit diesen Anmeldeinformationen eine Verbindung zu Ihrem WLAN-AP her. Navigieren Sie zu Info -> Netzwerk und suchen Sie im Abschnitt „Netzwerkschnittstelle (Station)“ nach der über DHCP erhaltenen IP-Adresse.
- Wenn Ihr WLAN-AP eine Zulassungsliste für MAC-Adressen verwendet, beachten Sie bitte, dass der ESP32 zwei verschiedene MAC-Adressen für seinen AP- und Client-Modus hat. Diese werden auch unter Info -> Netzwerk aufgeführt.
- Wenn OpenDTU mit einem konfigurierten WiFI-AP verbunden ist, wird der Access Point „OpenDTU-*“ nach 3 Minuten geschlossen.

**Reply by Kante001** [reply]
OK Problem 1 gelöst. Er darf nicht per USB am PC hängen zur Stromversorgung ich hab in an ein USB Netzteil gehangen damit konnte ich in das Open DTU netzwerk Rest hat so funktioniert.
Mir fällt gerade auf wenn ich am PC schaue auf der weboberfläche sehe ich die letzt Aktualisierung hochzählen so wie es soll. Gucke ich am Handy steht er Star bei einer Zahl und zählt die Zeit nicht weiter hoch. Vorher ging das immer woran liegt das. Jetzt ist die Frage ob deswegen die Nulleinspeisung nicht geht weil der raspery auch die Daten nicht richtig bekommt ?

**Reply by ghost** [reply]
Chrome als Browser?
Üblicherweise wird vom Browser versucht, sowenig wie möglich Daten zu übertragen und lieber auf den Cache zurückgegriffen.
Lösche mal bitte den Browsercache,

**Reply by Kante001** [reply]
Ja Chrom. Hab ich gelöscht aber hat nix gebracht. Komisch mit dem alten Datenstand ging das immer. Kann es sein das ich irgendwo ein falsches Kreuz in den Einstellungen habe ?aber wahrscheinlich nicht da es PC ja geht. Sonst ging das immer am Handy.

**Reply by Kante001** [reply]
Hab mir mal einen anderen Browser geladen bei Opera geht es mit dem aktualisieren. Woran kann das den jetzt wieder liegen das Chrom irgendwie nicht damit klar kommt.

**Reply by ghost** [reply]
Wenn es am PC geht, glaube ich nicht , das eine Einstellung an der DTU fehlerhaft ist.
Gute Frage.
Du meinst die Anzeige in Sekunden, wann sich die DTU das letzte Mal mit dem Wechselrichter verbunden hat?
Kommen überhaupt  Live-Daten?
In der Live Ansicht ist direkt hinter dem Text "Live-Daten" ein kleiner Punkt. Ist der grün animiert oder nur rot?
Das ist eine Art Lebenzeichen, bzw. zeigt an, das die Web App Verbindung zur DTU hat.

**Reply by ghost** [reply]
Zu deinen Raspi Problem.
Ich hab da 0 Ahnung. Bei mir lese ich nur den WR aus.
Möglicherweise hat sich in der Web API und beim MQTT auch einiges geändert, ich benutze das nicht.
Vielleicht kämpfst du dich [hier](https://www.opendtu.solar/firmware/) mal durch.

**Reply by ghost** [reply]
Liegt an Chrome.
Evtl. mal alles löschen. Verlauf, Cache ... Möglicherweise gibt es Einstellungen dazu in Chrome.
Chrome mal aktualisieren, falls es da was neues gibt.

**Reply by Kante001** [reply]
Chrom kein Update verfügbar. Hab schon alles gelöscht was ging in Chrom. Bei der alten Version hat er nach dem aktualisieren nicht dieses warte auf Daten (kann bis zu 10 sec dauern) nicht gemacht vielleicht liegt es bei Chrom daran wie er die Daten bekommt ? Ja Status ist grün. Irgendein Chrom Problem aber das müssten doch mehrere haben ?

**Reply by Kante001** [reply]
Hab gerade mal inkognito tab probiert da geht es in Chrom. Merkwürdig

**Reply by ghost** [reply]
Das "Warte auf Daten..." ist recht neu.
Trotzdem sollten die Sekunden hochzählen.
Mit Opera klappt es ja bei dir. Keine Ahnung.
Ich habe mit Chrome am iPhone und am Samsung glücklicherweise keine Probleme.

@Entwickler
@tbnobody
Eine Idee??

**Reply by Kante001** [reply]
OK habe gerade noch mal rausgefunden wenn ich die Open dtu aus den Favoriten lösche und danach den Cache leeren geht es. Sobald ich ihn als Favoriten speichere geht es nicht mehr.

**Reply by ghost** [reply]
Deine Geduld möchte ich haben.
Mir fehlen die Worte, und ein "Grinsen" sitzt mir auch im Gesicht.
Einen "blöden" Gedanken habe ich auch noch: Man könnte Google mal zu den Darstellungsproblemen befragen. Ob da eine Antwort kommt?

**Reply by ghost** [reply]
Das Problem, was ich hier sehe ist ziemlich speziell. 
Ob das Google Chrome Team helfen kann, glaube ich fast nicht.
Du wirst das wohl so hinnehmen müssen.
Du kannst nur hoffen, das sich noch jemand dazu meldet. Mir fällt erstmal nichts mehr ein.

**Reply by ghost** [reply]
Da das hier ziemlich lang ist, könnte es sein, das keiner dazu beiträgt.
Wenn die nächsten Tage nichts kommt, schließe  diese Diskussion und mache nochmal ein neues Thema auf.

**Reply by Kante001** [reply]
Also es muss irgendwas mit dem Cache oder so zu tun haben. Wenn ich paar websiten öffne Open dtu wieder schließe und den Browser wieder öffne geht auch nicht mehr liegt also nicht an den Favoriten. Inkognito geht es immer. Ja vielleicht hat noch jemand eine zündende Idee. Mal morgen abwarten ob die Steuerung wieder geht.
